src/modules/fancyzones/lib/util.cpp

```diff
@@ -161,5 +161,9 @@ bool IsInterestingWindow(HWND window, const std::vector<std::wstring>& excludedA
     {
         return false;
     }
+    if (find_app_name_in_path(filtered.process_path, { L"FANCYZONESEDITOR.EXE" }))
```

Comment:
We should not hard-code specific apps, we should understand why this was zoned and use a general rule.

Reply:
Every window that is _interesting_ window is zoned, reason why `FancyZonesEditor.exe` was interesting window previously is already wrong, but it worked because we didn't have this _open window in active monitor_ feature. For sure we should not consider `FancyZonesEditor.exe` for interesting window, same as we should not consider `PowerLaucher.exe` as interesign window (just line above this one). We can list these processes as const strings in the beginning of this file, just to have them in one place, and I'll make that change.

Reply:
For PowerLaucher.exe that was a quick hotfix, but there is an issue open to properly handle it setting the right flag on the PowerLaucher.exe window.

FancyZonesEditor.exe should not be moved to the primary monitor because it is opened on the secondary monitor and when a window is opened on a secondary monitor it should never be moved to the primary monitor (I think this is the same problem I was seeing with notepad, opening on the wrong monitor).

Reply:
Problem with notepad is actually behavior by notepad itself, that it opens in last known position, and it interfere with our custom positioning. I'll address that issue in new iteration, for the windows that already by itself want to open on currently active monitor, I'll just skip custom positioning.